dspace-api/src/main/java/org/dspace/ctask/general/MetadataWebServiceDataInfo.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.ctask.general;
+
+import javax.xml.xpath.XPathExpression;
+
+/**
```

Comment:
This class is an extraction of the inner classes from "MetadataWebService", I don't have a clue what this class represents. Just wanted to extract it so that the self assignment issue was gone & that you actually used getters/setters to set variables instead of public variables.